2019btechcse056.md

```diff
@@ -0,0 +1,5 @@
+Title 	           ||     Author 	      ||      Github 	     ||    Date    ||    Link 
```

Comment:
- Formatting not done correctly
- Please add at least 5 resources
- Add proper title and description at the beginning of the file
- Add metadata at the top
- Name your file appropriately instead of your roll no

Reply:
@uditagarwal1305 please do these few changes

Reply:
Refer to exampleFile for more information